modules/cnv-removing-interface-from-nodes.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-configuring-node-network-policy
+
+[id="cnv-removing-interface-from-nodes_{context}"]
+= Removing an interface from nodes
+
+Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object that declares the `state` of the interface to `absent`.
```

Comment:
This sentence is a bit ambiguous-sounding, IMO.

Reply:
Changed to:
"Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object and set
the `state` of the interface to `absent`."